CRM/Member/BAO/Membership.php

```diff
@@ -849,8 +849,10 @@ public static function getContactMembership($contactID, $memType, $isTest, $memb
       $params = array('id' => $memType);
       $membershipType = array();
       if (CRM_Member_BAO_MembershipType::retrieve($params, $membershipType)) {
-        $memberTypesSameParentOrg = CRM_Member_BAO_MembershipType::getMembershipTypesByOrg($membershipType['member_of_contact_id']);
-        $memberTypesSameParentOrgList = implode(',', array_keys($memberTypesSameParentOrg));
+        $memberTypesSameParentOrg = civicrm_api3('MembershipType', 'get', array(
```

Comment:
This does look like it will run into problems when there are more than 25 membership types due to API requests having a default limit of 25 values.

Reply:
arg true!

Reply:
@mattwire did you see that comment?

Reply:
:-(  Please see new PR #11029 that fixes this and also updates deprecated function based on your comments @eileenmcnaughton @tschuettler